staramr/blast/JobHandler.py

```diff
@@ -93,8 +94,24 @@ def run_blasts(self, files) -> None:
         db_files = self._make_db_from_input_files(self._input_genomes_tmp_dir, files)
         logger.debug("Done making blast databases for input files")
 
-        logger.info("Scheduling MLST for input files")
-        self._schedule_mlst(db_files)
+        future_mlst_db = [] # type: list
+
+        if self._thread_pool_executor is not None:
+          logger.info("Scheduling MLST for input files")
+          future_mlst_db.append(self._thread_pool_executor.submit(self._schedule_mlst, db_files))
+
+        try:
+            for future_mlst in future_mlst_db:
+                mlst_result = future_mlst.result()
+
+                if mlst_result is not None:
+                    self._mlst_data += mlst_result
+                else:
+                    self._mlst_data = mlst_result
```

Comment:
If `mlst_result` is `None` we should likely raise an exception as this should never occur. As it is written now, if the value was `None`, it would overwrite all previous `self._mlst_data` results.

Reply:
Decided to remove this logic since after testing it a few times there seems to no errors so far